internal/pg/pool.go

```diff
@@ -122,3 +131,16 @@ func (p *Pool) checkConnection(ctx context.Context) error {
 
 	return nil
 }
+
+// Currently supported locale variants, compromised between https://www.postgresql.org/docs/9.3/multibyte.html
+// and https://www.gnu.org/software/libc/manual/html_node/Locale-Names.html.
+//
+// Valid examples:
+// * en_US.utf8,
+// * en_US.utf-8
+// * en_US.UTF8,
+// * en_US.UTF-8
+func validUtf8Locale(setting string) bool {
+	match, _ := regexp.MatchString("en_US.(utf|UTF)-?8", setting)
```

Comment:
Agree, with compiling regex. If we go with lowercasing, it might pose an issue with `en_US` not being properly defined. Do you presume to know if `en_us` is also ok? If

Reply:
I _think_ that those locales a handled by libc, and it does normalization by lowercasing everything. But I'm not sure of it.

Reply:
I presume that's correct as `en_US.utf8` also works, despite database entries always seem to be uppercased.